src/core/tsi/ssl_transport_security.cc

```diff
@@ -1076,8 +1076,9 @@ static int CheckCertRevocation(grpc_core::experimental::CrlProvider* provider,
   // analogue for not finding the Crl from the provider, thus the certificate in
   // question is Undetermined.
   if (absl::IsNotFound(crl.status())) {
-    // TODO(gtcooke94) knob for undetermined being revoked or unrevoked. By
-    // default, unrevoked.
+    if (provider->DenyUndetermined()) {
+      return 0;
```

Comment:
In this case (and in the `else if` below): do we need to free `*crl`?

Reply:
I don't believe so - this is an absl::StatusOr, so in the case that it is an absl::Status, there won't be a CRL to free

Reply:
Yup good call, agreed.